other/dns-policy-and-dns-config/dns-policy-and-dns-config.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: dns-policy
+  annotations:
+    policies.kyverno.io/title: Change DNS Config and Policy
+    policies.kyverno.io/category: Other
+    policies.kyverno.io/severity: medium
+    kyverno.io/kyverno-version: 1.7.3
+    kyverno.io/kubernetes-version: "1.23"
+    policies.kyverno.io/subject: Pod
+    policies.kyverno.io/description: >-
+      The Default DNS policy in Kubernetes gives the flexibility of service 
+      access; however, it costs some latency on a high scale, and it needs to 
+      be optimized. This policy helps us to optimize the performance of DNS 
+      queries by setting DNS Options, nodelocalDNS IP, and search Domains. 
+spec:
+  rules:
+  - name: dns-policy
+    context:
+    - name: dictionary
+      configMap:
+        # kubelet-config cm would also works by using clusterDomain 
+        # instead of clusterName; but kubeadm-config sounds more reliable
+        # when considering kubelet-config is changed every cluster upgrade, etc.
+        name: kubeadm-config 
+        namespace: kube-system
+    match:
+      resources:
```

Comment:
Please add an `any` or `all` block under `match` here to use the latest policy definition language.

Reply:
It's done, I added `any` block.